{{ cookiecutter.repo_name }}/MANIFEST.in

```diff
@@ -1,14 +1,11 @@
-include AUTHORS.rst
-include LICENSE
-include README.rst
-include requirements.txt
+graft src
+graft tests
+graft requirements
 
-recursive-exclude * __pycache__
-recursive-exclude * *.py[co]
+include AUTHORS.txt LICENSE*.txt README.rst
 
-recursive-include docs *.rst conf.py Makefile make.bat
-
-include {{ cookiecutter.package_dir_name }}/version.py
-
-# If including data files in the package, add them like:
-# include path/to/data_file
+# Exclude all bytecode files and __pycache__ directories
+global-exclude *.py[cod]  # Exclude all .pyc, .pyo, and .pyd files.
+global-exclude .DS_Store  # Exclude Mac filesystem artifacts.
```

Comment:
shall we also exclude `.idea` which comes from our pycharm users?

Reply:
Yes, included